src/ui/components/search/searchcomponent.js

```diff
@@ -214,11 +214,6 @@ export default class SearchComponent extends Component {
 
   onCreate () {
     if (this.query != null && !this.redirectUrl) {
-      this.core.persistentStorage.set(
-        StorageKeys.REFERRER_PAGE_URL,
-        this.core.globalStorage.getState(StorageKeys.REFERRER_PAGE_URL),
-        true
-      );
```

Comment:
Is this logic handled in core now?

Reply:
Yes, this was to band-aid the problem of not being able to batch different replaceHistory calls. It was essentially setting with replaceHistory=true before it ever got to a batch that had replaceHistory=false.